pkg/gen/gen_resources_cmds.go

```diff
@@ -306,33 +320,36 @@ func addToTemplateData(data *TemplateData, specVersion SpecVersion, nsName, resN
 			return nil
 		}
 
-		properties := getMethodProperties(specVersion, specOp, op)
+		properties, enums := getMethodProperties(specVersion, specOp, op)
 
 		if hasSubResources {
 			data.Versions[specVersion].Namespaces[nsName].Resources[resCmdName].SubResources[subResCmdName].Operations[op.MethodName] = &OperationData{
 				Path:      op.Path,
 				HTTPVerb:  httpString,
 				PropFlags: properties,
+				EnumFlags: enums,
 			}
 		} else {
 			data.Versions[specVersion].Namespaces[nsName].Resources[resCmdName].Operations[op.MethodName] = &OperationData{
 				Path:      op.Path,
 				HTTPVerb:  httpString,
 				PropFlags: properties,
+				EnumFlags: enums,
 			}
 		}
 	}
 
 	return nil
 }
 
-func getMethodProperties(specVersion SpecVersion, specOp *spec.Operation, op spec.StripeOperation) map[string]string {
+func getMethodProperties(specVersion SpecVersion, specOp *spec.Operation, op spec.StripeOperation) (map[string]string, map[string][]spec.StripeEnumValue) {
```

Comment:
non blocking nit: could probably rename this method now to make it clearer

Reply:
Ah yea. I couldn't think of a better name but open to ideas 😅